tests/integration/hive_metastore/test_migrate.py

```diff
@@ -101,3 +102,44 @@ def test_migrate_external_table(ws, sql_backend, inventory_schema, make_catalog,
 
     target_tables = list(sql_backend.fetch(f"SHOW TABLES IN {dst_schema.full_name}"))
     assert len(target_tables) == 1
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=5))
+def test_revert_migrated_table(ws, sql_backend, inventory_schema, make_schema, make_table):
+    schema1 = make_schema(catalog_name="hive_metastore")
+    schema2 = make_schema(catalog_name="hive_metastore")
+    tables = [
+        make_table(schema_name=schema1.name),
+        make_table(schema_name=schema1.name),
+        make_table(schema_name=schema2.name),
+        make_table(schema_name=schema2.name),
+    ]
+
+    for table in tables[0:3]:
+        # apply "upgraded to" table property to the first three tables.
```

Comment:
Why?

Reply:
I want the demonstrate the following use cases:
1. Table was migrated and need to be reverted
2. Table was migrated and no being reverted
3. Table was not migrated

Reply:
Fixed